gtsfm/frontend/matcher/dummy_matcher.py

```diff
@@ -13,21 +14,25 @@ class DummyMatcher(MatcherBase):
 
     def match(
         self,
-        descriptors_im1: np.ndarray,
-        descriptors_im2: np.ndarray,
-        distance_type: MatchingDistanceType = MatchingDistanceType.EUCLIDEAN,
+        keypoints_i1: Keypoints,  # pylint: disable=unused-argument
+        keypoints_i2: Keypoints,  # pylint: disable=unused-argument
+        descriptors_i1: np.ndarray,
+        descriptors_i2: np.ndarray,
+        distance_type: MatchingDistanceType = MatchingDistanceType.EUCLIDEAN,  # pylint: disable=unused-argument
```

Comment:
thanks for the updates @ayushbaid . should we make a new `MatchingDistanceType.LEARNED`?

Reply:
Interesting idea. Technically, we are not providing this option to the user for learned matchers. I think this should not be a property of the base matcher class?

Reply:
Irrelevant now because I have removed the distance type from the base matcher.